src/grimoirelab/core/config/settings.py

```diff
@@ -96,6 +96,7 @@
     'rest_framework',
     'grimoirelab.core.scheduler',
     'grimoirelab.core.scheduler.tasks',
+    'grimoirelab.core.scheduler.datasources',
```

Comment:
I meant the datasources should be at the same level of scheduler, not inside.

`grimoirelab.core.datasources`